pkg/controller/config/config_controller.go

```diff
@@ -20,6 +19,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 
+	"github.com/Juniper/contrail-operator/pkg/controller/utils"
```

Comment:
Shouldn't all imports from contrail-operator be in one block?

Reply:
This was fixed automatically by goimports

Reply:
done